docker-compose/run_integration_tests_local_keybase.sh

```diff
@@ -0,0 +1,199 @@
+  echo_msg "ERROR: The cert file could NOT be found at: ${keybase_cert_file}"
+  exit 1
+fi
+if [ -f ${keybase_key_file} ]
+then
+  echo_msg
+  echo_msg "  - OK: The key file was found at: ${keybase_key_file}"
+else
+  echo_msg
+  echo_msg "ERROR: The key file could NOT be found at: ${keybase_key_file}"
+  exit 1
+fi
+
+# Copy certfiles from KeyBase to local for container mount
+echo_msg "  - COPY certfiles from KeyBase to local temp for container mount..."
+echo_msg
+cp ${keybase_cert_file} "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.cert.pem"
+cp ${keybase_key_file} "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.key.nocrypt.pem"
+
+# Check if tests are to run in docker-compose
+if [[ $1 == "dc" ]]
+then
+  # Stop web if running and restart db
+  docker-compose stop web
+  docker-compose restart db
+
+  # Run docker-compose containter one-off
+  echo_msg
+  echo_msg "------RUNNING DOCKER-COMPOSE CONTAINER WITH INTEGRATION TESTS------"
+  echo_msg
+  echo_msg "    INTEGRATION_TESTS_LIST: ${INTEGRATION_TESTS_LIST}"
+  echo_msg
+  docker-compose run \
+    --service-ports \
+    -e HOSTNAME_URL=${HOSTNAME_URL} \
+    -e DJANGO_USER_ID_SALT=${DJANGO_USER_ID_SALT} \
+    -e DJANGO_USER_ID_ITERATIONS=${DJANGO_USER_ID_ITERATIONS} \
+    -e DJANGO_FHIR_CERTSTORE=${DJANGO_FHIR_CERTSTORE} \
+    -v "${CERTSTORE_TEMPORARY_MOUNT_PATH}:${DJANGO_FHIR_CERTSTORE}" \
+    web bash -c "python runtests.py --integration ${INTEGRATION_TESTS_LIST}"
+fi
+
+# Check if tests are to run in docker CBC container one-off
+if [[ $1 == "cbc" ]]
+then
+  echo_msg
+  echo_msg "------RUNNING DOCKER CONTAINER CBC IMAGE WITH INTEGRATION TESTS------"
+  echo_msg
+  echo_msg "    INTEGRATION_TESTS_LIST: ${INTEGRATION_TESTS_LIST}"
+  echo_msg
+  docker run \
+    -e CERT_FILE=${CERT_FILE} \
+    -e KEY_FILE=${KEY_FILE} \
+    -e DJANGO_USER_ID_SALT=${DJANGO_USER_ID_SALT} \
+    -e DJANGO_USER_ID_ITERATIONS=${DJANGO_USER_ID_ITERATIONS} \
+    -e DJANGO_FHIR_CERTSTORE=${DJANGO_FHIR_CERTSTORE} \
+    --env-file ${keybase_env} \
+    --mount type=bind,source="$(pwd)",target=/app,readonly \
+    --mount type=bind,source="${CERTSTORE_TEMPORARY_MOUNT_PATH}",target=/certstore,readonly \
+    --rm \
+    ${DOCKER_IMAGE}:${DOCKER_TAG} bash -c "cd /app; \
+                                           pip install -r requirements/requirements.txt \
+                                                        --no-index --find-links ./vendor/; \
+                                           pip install sqlparse; \
+                                           python runtests.py --integration ${INTEGRATION_TESTS_LIST}"
+fi
+
+# Remove certfiles from local directory
+echo_msg
+echo_msg Shred and Remove certfiles from CERTSTORE_TEMPORARY_MOUNT_PATH=${CERTSTORE_TEMPORARY_MOUNT_PATH}
+echo_msg
+shred "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.cert.pem"
+rm "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.cert.pem"
+shred "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.key.nocrypt.pem"
+rm "${CERTSTORE_TEMPORARY_MOUNT_PATH}/ca.key.nocrypt.pem"
```

Comment:
I'm not sure how commonly installed shred is, but if you add `set -euo pipefail` above you'll need to add some if/then/else logic here so it won't fail if shred isn't installed.

Reply:
@njdister Good call. Shred isn't a builtin, so is good to check for it. I just updated the script to check for it. TY!